lib/core.js

```diff
@@ -206,6 +207,19 @@ function Core(options) {
 
           GlobalConfig.setKeepAlive(keepAlive);
         }
+
+        // const credentialCacheDir = options.credentialCacheDir;
+
+        const customCredentialManager = options.customCredentialManager;
+        if (Util.exists(customCredentialManager)) {
+          Errors.checkArgumentValid(Util.isObject(customCredentialManager),
+            ErrorCodes.ERR_GLOBAL_CONFIGURE_INVALID_CUSTOM_CREDENTIAL_MANAGER);
+
+          GlobalConfig.setCustomCredentialManager(customCredentialManager);
+        } 
+        // else {
+        //   GlobalConfig.setCustomCredentialManager(new JsonCrednetialManager(credentialCacheDir));
+        // }
```

Comment:
Please remove unused code

Reply:
Sorry, I was waiting for the confirmation. I wasn't sure whether I could move the cachedir option to the connection config or not because the conversation above was not resolved yet.  I will remove this.

Reply:
I removed it.